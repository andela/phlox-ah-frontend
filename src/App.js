import React, { Component } from 'react';
// eslint-disable-next-line
import { render } from 'react-dom';
import Home from './containers/Home/Home';

/**
 *
 *
 * @class App
 * @extends {Component}
 */
class App extends Component {
  /**
   * @description - This method renders the jsx for this component
   * @returns {jsx} -
   */
  // eslint-disable-next-line
  render() {
    return (
      <div>
        <h1 className="theme-color">Hello! Welcome to Authors Haven</h1>
        This is the app component
        <Home />
      </div>
    );
  }
}

export default App;
