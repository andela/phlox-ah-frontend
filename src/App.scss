* {
  padding: 0;
  margin: 0;
  box-sizing: border-box;
}

a{
  color: inherit;
}
body { font-size: 18px; }
