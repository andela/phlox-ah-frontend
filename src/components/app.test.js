import React from 'react';
import renderer from 'react-test-renderer';
import App from './app';

test('<App/>', () => {
  const component = renderer.create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();// eslint-disable-line 
});
