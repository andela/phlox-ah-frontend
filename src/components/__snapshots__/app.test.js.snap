// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> 1`] = `
<div>
  <h1>
    Hello! Welcome to Authors Haven
  </h1>
</div>
`;
